synapse/models/inet.py

```diff
@@ -325,7 +327,6 @@ def initCoreModule(self):
         self.core.addTypeCast('inet:ipv4:cidr', ipv4cidr)
         self.onFormNode('inet:fqdn', self.onTufoFormFqdn)
         self.onFormNode('inet:passwd', self.onTufoFormPasswd)
-        self.revCoreModl()
```

Comment:
Will removing these revCoreModl calls result in ending up on different versions?

Reply:
nope.  just handled differently.